time_functions.py

```diff
@@ -0,0 +1,28 @@
+def month_add(month, offset):
+    """ month_add takes two parameters: month = (1 <= month <= 12); offset(-infinity <= offset <= infinity)
+        return value: 1 <= result <= 12
+
+        Take a month and an offset and add them together with appropriate wraparound behavior
+        for both positive and negative offset values.
+    """
+    if month < 1 or month > 12:
+        return month;#return bad value and hopefully have it be noticed.
```

Comment:
We should handle this by raising an exception, instead. Exceptions tell the programmer they're doing it wrong.